projects/sandbox/export/pyproject.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Alec Gunny <alec.gunny@gmail.com>"]
 
 
 [tool.poetry.scripts]
-export-model = "export:export"
+export = "export:export"

```

Comment:
I would be careful with this, bash already reserves `export` and as I recall with DeepClean it used to complain when I originally called the script this

Reply:
Ah, okay. I'll change the call in the pyprojec.toml then.